### ISSUE TITLE
Image rotation using Fabulous.XamarinForms.FFImageLoading?

### ISSUE THREAD

**Issue by gaelian** [fsprojects/Fabulous#870]
## Question / Discussion

I'm trying to translate this C# code for use with Fabulous.XamarinForms.FFImageLoading:
```c#
var t = ImageService.Instance.LoadStream(async c => await image.OpenReadAsync()); // image is a Xamarin.Essentials.FileResult.
t.Transform(new RotateTransformation(90));
using var imageStream = await t.AsPNGStreamAsync();
```
I've gotten this far in F#:
```f#
let t = FFImageLoading.ImageService.Instance.LoadStream(fun _ -> image.OpenReadAsync())
t.Transform(FFImageLoading.Transformations.RotateTransformation(90.)) |> ignore
use! imageStream = t.AsPNGStreamAsync() |> Async.AwaitTask
```
But in my F# code the `AsPNGStreamAsync` extension method cannot be found.

I also don't understand why `open Fabulous.XamarinForms.FFImageLoading` or `open FFImageLoading` both don't seem to work and I need to use use `FFImageLoading.ImageService ...` and `FFImageLoading.Transformations ...` instead. But I suspect this may have something to do with my problem.

Can anyone tell me what I'm doing wrong?

**Comment by Happypig375** [comment]
AsPNGStreamAsync is only defined in platform-specific versions. https://github.com/luberda-molinet/FFImageLoading/search?q=aspngstreamasync

Are you targeting .NET Standard?

**Comment by gaelian** [comment]
Yes. I’ve been trying to do this in the shared project of a Xamarin Forms app.

**Comment by Happypig375** [comment]
Then you should try this in platform-specific projects.

**Comment by vshapenko** [comment]
Glide and Nuke would be better for such cases.
